File: app/src/main/java/niks/com/mseapp/SplashScreen_Menu.java
package niks.com.mseapp;

import android.content.DialogInterface;
import android.content.Intent;
import android.content.SharedPreferences;
import android.os.Bundle;
import android.support.v7.app.AlertDialog;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.ImageButton;

import java.util.Timer;
import java.util.TimerTask;

public class SplashScreen_Menu extends AppCompatActivity {

   ImageButton screen1, screen2;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.splashscreen_menu);


        screen1 = (ImageButton)findViewById(R.id.screen1);
        screen2 = (ImageButton)findViewById(R.id.screen2);


        screen1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                Intent nik = new Intent(getApplicationContext(), MainActivity2.class);
                startActivity(nik);
                finish();

            }
        });


        screen2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                Intent nik = new Intent(getApplicationContext(), MainActivity3.class);
                startActivity(nik);
                finish();

            }
        });


    }//oncreate

    @Override
    public void onBackPressed() {

                new AlertDialog.Builder(SplashScreen_Menu.this)
                .setMessage("Do you want to exit!")
                .setIcon(R.mipmap.ando)
                .setPositiveButton("YES", new DialogInterface.OnClickListener() {
                    @Override
                    public void onClick(DialogInterface dialog, int which) {

                        Intent homeIntent = new Intent(Intent.ACTION_MAIN);
                        homeIntent.addCategory(Intent.CATEGORY_HOME);
                        homeIntent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
                        startActivity(homeIntent);
                        finish();



                    }
                }).setNegativeButton("NO", new DialogInterface.OnClickListener() {
            @Override
            public void onClick(DialogInterface dialog, int which) {
                dialog.cancel();
            }
        }).show();


    }


}//main

File: app/src/main/java/niks/com/mseapp/activities/SplashScreen_Menu.kt
package niks.com.mseapp.activities

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AlertDialog
import android.support.v7.app.AppCompatActivity
import android.view.View
import android.widget.ImageButton
import niks.com.mseapp.R

class SplashScreen_Menu : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.splashscreen_menu)


        val screen1 = findViewById<View>(R.id.screen1) as ImageButton
        val screen2 = findViewById<View>(R.id.screen2) as ImageButton


        screen1.setOnClickListener {
            val nik = Intent(applicationContext, MainActivity2::class.java)
            startActivity(nik)
            finish()
        }


        screen2.setOnClickListener {
            val nik = Intent(applicationContext, MainActivity3::class.java)
            startActivity(nik)
            finish()
        }


    }//oncreate

    override fun onBackPressed() {

        AlertDialog.Builder(this@SplashScreen_Menu)
                .setMessage("Do you want to exit!")
                .setIcon(R.mipmap.ando)
                .setPositiveButton("YES") { dialog, which ->
                    val homeIntent = Intent(Intent.ACTION_MAIN)
                    homeIntent.addCategory(Intent.CATEGORY_HOME)
                    homeIntent.flags = Intent.FLAG_ACTIVITY_CLEAR_TOP
                    startActivity(homeIntent)
                    finish()
                }.setNegativeButton("NO") { dialog, which -> dialog.cancel() }.show()


    }


}//main
